### PR TITLE
Add GitHub blogs

### PR DESCRIPTION
**Pull request by lauravuo**

### CODE REVIEW

File: content/en/blog/github-swiss-knife-3/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+The CI runs tests on Dependabot PRs similar to those, as with the changes developers have made.
+One can review and merge the PR similarly.
+
+Using Dependabot requires a comprehensive enough automated test set. The one who merges
+the PR needs to confirm that the change will not cause regression. Manual testing is required
+if the CI is missing the automated tests for the changes. If one needs manual testing, the benefit
+of using the bot for automated PR creation is lost, and it might even be more laborious to update
+the dependencies one by one.
+
+In our project, we used the principle of testing all the crucial functionality with
```
Comment (lainio) [inline content/en/blog/github-swiss-knife-3/index.md:77]: The chapter is well-written and vital. Should it be highlighted somehow? Unfortunately, nothing came to my mind. Maybe learning snippet to the end: 

> You should estimate the risk/effort ratio when automating crucial tasks.

File: content/en/blog/github-swiss-knife-3/index.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  },
+  "postCreateCommand": "asdf direnv setup --shell bash --version latest"
+}
+```
+
+[The GitHub CLI](https://cli.github.com/) tool was another helpful tool we found during the agency project.
+It offers the GitHub web application functionality via the command line. The tool is especially useful
+for those who suffer from context switching
+from terminal to browser and vice versa or who prefer to work from a terminal altogether.
+In addition, the CLI tool can be used to automate certain routines, specifically in CI.
```
Comment (lainio) [inline content/en/blog/github-swiss-knife-3/index.md:181]: Other cons of `gh` are that it provides a convenient playground for GitHub APIs and lets you play with JSON payloads quickly. All of that provides an excellent platform to build IDE/PDE add-ons. 

In `find-network` projects, the `Makefile` rule implementation has been a great benefit -- personally, at least.

**Review by lainio — APPROVED** [review]
Good stuff!

**Comment by lauravuo** [comment]
Thanks for the review 👍 Great comments.

**Pull request merged**